translations/messages-en.properties

```diff
@@ -216,6 +216,9 @@ analytics.reporting.health_centers_size = SMO
 analytics.reporting.clinics_size = Reporting Unit
 analytics.reporting.not_submitted = Reports Missing
 analytics.reporting.incomplete = Invalid Reports
+analytics.reporting.change_time_unit = Change time unit
+analytics.reporting.last_time_unit = Last {{time_unit}}
+analytics.reporting.last_time_unit.plural = Last {{quantity}} {{time_unit}}s
```

Comment:
This key will only work in English. The pluralisation of time_unit has to be a separate translation. I think we need something like:

```
week = Week
week.plural = Weeks
analytics.reporting.last_time_unit = Last {{quantity}} {{time_unit}}
```

Then you can translate `analytics.reporting.last_time_unit` passing in the translated value for `week` or `week.plural`.